### ISSUE TITLE
For `toolbar top` component, announce the title first then move focus to the first element

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-ios#1584]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-ios/issues) for duplicate or closed feature requests

### Your test device

iPhone 17

### Device OS version

26.5

### Is Liquid Glass disabled

- [ ] Yes, Liquid Glass is disabled (or OS version is older than 26)
- [ ] No, Liquid Glass is enabled (or OS version is 26 or newer)

### OUDS iOS library version

2.2.0

### Describe the issue

When new screen is displayed, the title is not read by default and focus is set to an inappropriate component

By default Voice Over set the focus to the next element after the title (not always consistent, and on some native settings screens it's not consistent at all), this is not very easy for users who may miss informations on the top.

### Expected Behavior

When navigate with screen reader, announce the screen title on display (no focus move), then move the focus to the first component on the page (often the back button), so the user will start from the top of the page and will not loose any information in reading order

### Accessibility settings

Voice Over
